package.json

```
@@ -38,7 +38,7 @@
     "chalk": "^4.1.2",
     "conventional-changelog-cli": "^2.1.1",
     "cross-env": "^7.0.3",
-    "esbuild": "^0.13.12",
+    "esbuild": "~0.14.0",
```

```suggestion
    "esbuild": "^0.14.0",
```

When the major version is 0, minor updates will be treated as breaking changes. So `^0.14.0` won't covers `^0.15.x` and it should be fine